code/modules/projectiles/guns/projectile.dm

```diff
@@ -195,28 +197,31 @@
 		to_chat(user, "<span class='warning'>[src] is empty.</span>")
 	update_icon()
 
-/obj/item/gun/projectile/attackby(var/obj/item/A as obj, mob/user as mob)
+/obj/item/gun/projectile/attackby(obj/item/A, mob/user)
 	..()
 	load_ammo(A, user)
 
-/obj/item/gun/projectile/attack_self(mob/user as mob)
-	if(is_jammed)
-		to_chat(user, "<span class='notice'>\The [user] unjams \the [src]!</span>")
-		if(do_after(user, 5))
-			playsound(src.loc, 'sound/weapons/empty.ogg', 100, 1)
-			is_jammed = 0
+/obj/item/gun/projectile/attack_self(mob/user)
+	if(jam_num)
+		playsound(src.loc, 'sound/weapons/empty.ogg', 50, 1)
+		jam_num--
```

Comment:
Maybe use the cooldown here too? Could get spammy.

Reply:
I tested it in-game, the volume is rather low, and I find the "click-click-clack" to be rather satisfying.